src/UmbracoExamine.PDF/Properties/AssemblyInfo.cs

```diff
@@ -1,29 +0,0 @@
-﻿using System.Reflection;
```

Comment:
I think the build script probably deleted this by accident. Have seen that happen when the file is locked (i.e. when VS is open and the build script is run at the same time)